web/src/main/java/se/inera/intyg/intygstjanst/web/integration/testability/TestabilityController.java

```diff
@@ -49,4 +51,17 @@ public Response createDefaultTestData() {
         testabilityService.createDefaultTestData();
         return Response.ok(VERIFICATION_MESSAGE).build();
     }
+
+    @PrometheusTimeMethod
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/createSickLeave")
+    public Response createSickLeave(CreateSickLeaveRequestDTO createSickLeaveRequestDTO) {
```

Comment:
Suggest that you return a CreateSickLeaveResponseDTO. It should include the certificateId that was created.